src/operator/concat.cc

```diff
@@ -46,29 +46,39 @@ Operator* ConcatProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shap
 DMLC_REGISTER_PARAMETER(ConcatParam);
 
 MXNET_REGISTER_OP_PROPERTY(Concat, ConcatProp)
-.describe(R"code(Concate a list of array along a given axis.
+.describe(R"code(Join input arrays along the given dimension.
 
-The dimension sizes of the input arrays on the given axis should be the same.
+.. note:: `Concat` is deprecated use `concat`.
 
-For example::
+The dimension of the input arrays should be the same except along the dimension
+which they are concatenated.
+The dimension of the output array along the concatenated dimension will be the
```

Comment:
Fix to: 

"The length of the output array along the concatenated dimension will be equal to the sum of the corresponding dimensions of the input arrays."

note: "input arrays", not "input array". Also dimension along dimension was confusing. Thus my preference for "length"